cfde_deriva/configs/portal/c2m2-level1-portal-model.json

```diff
@@ -2479,7 +2673,39 @@
             ],
             "missingValues": [ "" ],
             "primaryKey": "id"
-         }
+         },
+         "deriva": {
+            "source_definitions": {
+               "columns": true,
+               "fkeys": true,
+               "sources": {
+                  "S_collection": {
+                     "markdown_name": "Collection",
+                     "comment": "List of collections referencing this subject granularity.",
+                     "source": [
+                         {"inbound": ["CFDE", "collection_subject_granularity_granularity_fkey"]},
+                         {"outbound": ["CFDE", "collection_subject_granularity_collection_fkey"]},
+                         "RID"
+                     ]
+                  },
+                  "S_file": {
+                     "markdown_name": "File",
+                     "comment": "List of files referencing this subject granularity.",
+                     "source": [
+                         {"inbound": ["CFDE",  "file_subject_granularity_granularity_fkey"]},
+                         {"outbound": ["CFDE",  "file_subject_granularity_file_fkey"]},
+                         "RID"
+                     ]
+                  }
+                }
+             },
+             "visible_foreign_keys": {
+                "*": [
+                   {"sourcekey": "S_collection"},
```

Comment:
In retrospect, it is funny that this wouldn't include a source for "List of subjects referencing this granularity" since the others are indirected versions of this (collections or files describing those subjects)

Reply:
Added subject table to the source definitions and visible foreign keys annotations.